inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -90,4 +90,12 @@ install(PROGRAMS src/openvino/__init__.py
         DESTINATION ${PYTHON_BRIDGE_CPACK_PATH}/${PYTHON_VERSION}/openvino
         COMPONENT ${PYTHON_VERSION})
 
-ie_cpack(${PYTHON_VERSION})
+# install Python samples
+
+ie_cpack_add_component(python_samples)
+
+install(DIRECTORY sample/
+        DESTINATION ${IE_CPACK_IE_DIR}/samples/python
+        COMPONENT python_samples)
+
+ie_cpack(${PYTHON_VERSION} python_samples)
```

Comment:
I suppose that if we build multiple version of python, the sample will be installed multiple times as well (and to the same location). Is it expected behavior? Looks strange if yes, because samples don't depend on python version..

Reply:
Actually presented only once in artifact directory cpack: \openvino_ci\private_builds\dldt\master\pre_commit\c96c000ac0dd82fe59c4000ee6eb07c941e9cb97\private_linux_ubuntu_18_04_release\cpack\inference-engine-2021.4.0-3261-c96c000ac0d-refs_pull_5406_head-Linux-python_tools.zip